spec/page_util_spec.rb

```diff
@@ -44,12 +49,23 @@
 
   let(:any_string) { /.*/ }
 
-  let(:page_util_instance) do
-    page_one = Page.new(filename: filename_1, structure: structure_1)
-    page_two = Page.new(filename: filename_2, structure: structure_2)
+  let(:resource) do
+    attributes = [attr_1, attr_2, Attribute.new(key: 'bill', value: 'test 3', is_translatable: false)]
+
+    pages = [Page.new(filename: filename_1, structure: structure_1, position: 1),
+             Page.new(filename: filename_2, structure: structure_2, position: 2)]
 
-    resource = Resource.new(pages: [page_one, page_two], onesky_project_id: 1)
+    Resource.create!(pages: pages,
```

Comment:
This would be a good time to use the `factory_girl` library: https://github.com/thoughtbot/factory_girl

I should probably have brought that up in our earlier conversation about mocks. For the times when a mock won't do and you need a real object, factory_girl provides a nice organized way to build those 'fixtures'. Not saying you have to introduce that at this point in the project (depending on how pressed you are for time), but it's what we use on other projects.